src/test/java/guitests/PanelInfoEmptyTest.java

```diff
@@ -0,0 +1,40 @@
+package guitests;
+
+import org.junit.Test;
+import org.loadui.testfx.utils.FXTestUtils;
+
+import prefs.Preferences;
+import prefs.PanelInfo;
+
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+public class PanelInfoEmptyTest extends UITest {
```

Comment:
Can use a more self-explanatory class name? If the name is not self-explanatory enough, add a header comment to the class.